index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      type="module"
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"
+    ></script>
+    <script
+      nomodule
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"
+    ></script>
+    <title>Jahongir Yusupov -- Profile</title>
+  </head>
+  <body>
+    <nav class="toolbar">
+      <ul>
+        <li>
+          <p class="logo-name">Yusupov</p>
+        </li>
+        <li>
+          <button class="menu-button">
+            <ion-icon name="menu-outline"></ion-icon>
+          </button>
```

Comment:
- [ ] Kindly make sure that your hamburger button has a `<nav>` tag, this is because the hamburger will have nav elements in it in future milestone and it ensure you follow best practice.